src/Standards/Generic/Tests/CodeAnalysis/UselessOverridingMethodUnitTest.1.inc

```diff
@@ -64,6 +64,17 @@ class Bar {
     public function sameNumberDifferentParameters($a, $b) {
         return parent::sameNumberDifferentParameters($this->prop[$a], $this->{$b});
     }
+
+    public function differentCase() {
+        return parent::DIFFERENTcase();
+    }
```

Comment:
This also needs a test to ensure that method names using non-ASCII characters in different case do not compare as the same name.

Your change should already handle this correctly, but it should still be safeguarded by a test.

Reply:
@jrfnl, I'm inclined to add this new test right below the other case-related test, but this means updating the line of a few tests in `UselessOverridingMethodUnitTest.php`, which will make reviewing the commit a bit harder, and I remember you already asked me once to add new tests to the end of the file. So I wanted to check with you before proceeding as, in this case, fewer lines will be updated and are lines that did not exist before this PR. Maybe I'm obsessing too much about keeping related tests together in the test file?

Reply:
@rodrigoprimo You'd only be updating the line numbers of tests being added in this PR, not of pre-existing tests, so all good, though it would be nice if the commits could stay atomic, i.e. the new test would be added to commit 1 and the other commits would be rebased on top of the adjusted first commit and get the line number update in their proper commit.

Reply:
Note: feel free to make the update in a separate commit though, I can always fix that up before I merge the PR.

Reply:
I'm happy either way. Whatever makes reviewing and merging the PR easier for you. In this particular case, I added the new test to the original commit to fix the conflicts with the subsequent commits.

Reply:
@rodrigoprimo I'm looking at the new test now, but it is not actually testing what should be tested...

> method names using non-ASCII characters in a different case do not compare as the same name.

```php
    public function differentCaseWithNonAnsiiCharacters() {
        return parent::DIFFERENTcaseWithNonAnsiiCharáctêrs();
    }
```

👆🏻 This test has different characters (ASCII characters replaced by non-ASCII), which makes this a different method name anyway, which would never be flagged.

```php
    public function Déjàvü() {
        return parent::déJàVü(); // This should be flagged, only ASCII chars have changed case.
    }

    public function Déjàvü() {
        return parent::DÉJÀVÜ(); // This should not be flagged as non-ASCII chars have changed case, making this a different method name.
    }
```

Does that help ?

Reply:
Apologies for missing this the first time I wrote the test. It should be fixed now.